diamm/helpers/solr_pagination.py

```diff
@@ -148,6 +144,16 @@ def type_list(self):
         return OrderedDict(filtered_facets)
 
     @property
+    def images_count(self):
+        image_count = self.result.facets['facet_fields']['public_images_b']
+        if not image_count:
+            return 0
+        i = image_count.index('true') + 1 if 'true' in image_count else None
```

Comment:
image_count is a list [false, 1, true, 2] so I get the index of true and it's value is at + 1